### PR TITLE
database entry for player

### PR DESCRIPTION
**Pull request by three0s**
add basic player data
need to merge 2 dbs maybe
print/dump for structs
update ozo

### CODE REVIEW

File: schemas/niba_player.sql [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+--
+-- Name: player_character player_character_id_fkey; Type: FK CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.player_character
+    ADD CONSTRAINT player_character_id_fkey FOREIGN KEY (id) REFERENCES public.user_id(id) ON UPDATE CASCADE ON DELETE CASCADE;
+
+
+--
+-- PostgreSQL database dump complete
```
Comment (joey-xu) [inline schemas/niba_player.sql]: Can you change the table names in niba_static.sql to English?

File: server/db_accessor.cpp [hunk 2]
```diff
@@ -134,19 +132,52 @@ bool db_accessor::create_user(boost::asio::yield_context &yield, const std::stri
...
+        ozo::into(characters), yield[ec]);
+    if (ec) {
+        BOOST_LOG_SEV(logger_, sev::error) << ec.message() << " | " << ozo::error_message(conn)
+                                           << " | " << ozo::get_error_context(conn);
+        return {};
+    }
+    if (characters.size() == 0)
         return {};
-    return iter->second;
+    auto &[name, gender, strength, dexterity, physique, spirit] = characters.at(0);
```
Comment (joey-xu) [inline server/db_accessor.cpp]: From my understanding, one user can have only one character. Correct?

File: server/db_accessor.cpp [hunk 3]
```diff
@@ -134,19 +132,52 @@ bool db_accessor::create_user(boost::asio::yield_context &yield, const std::stri
...
+            .strength = strength, .dexterity = dexterity, .physique = physique, .spirit = spirit}};
 }
 
-bool db_accessor::create_char(const std::string &id, nibashared::character &&character) {
-    auto c = get_char(id);
-    if (c) {
+bool db_accessor::create_char(const std::string &id, const nibashared::player &player,
+                              boost::asio::yield_context &yield) {
+    ozo::error_code ec{};
+    // the insert string is unfortunately long and hand written
```
Comment (joey-xu) [inline server/db_accessor.cpp]: Static data will be longer.

Comment (three0s) [reply server/db_accessor.cpp]: there are json dump views set up, you can leverage that

File: server/db_accessor.cpp (hunk 2 above)
Comment (three0s) [reply server/db_accessor.cpp]: yes

**Pull request merged**